sdk/storage/Azure.Storage.DataMovement.Files.Shares/BlobToFileSharesTests/ClientBuilderExtensions.cs

```diff
@@ -21,6 +21,8 @@
 using Azure.Storage.Blobs.Models;
 using Azure.Storage.Test;
 using DMBlob::Azure.Storage.DataMovement.Blobs;
+using Azure.Core;
+using Azure.Core.TestFramework;
```

Comment:
Only change in the file?

Reply:
Good catch, will remove this. I was trying to do some fun stuff with SAS and decided not to do it here.